.env.example

```diff
@@ -1,5 +1,5 @@
-# Base Path Openshift: businesses/create Firebase:
-VUE_APP_PATH=businesses/create
+# Base Path Openshift: /businesses/create Firebase: /
+VUE_APP_PATH=/businesses/create
 
 #vaults Shared
 VUE_APP_ADDRESS_COMPLETE_KEY=
```

Comment:
Are you going to update lines 9 and 10 below?

Reply:
Yeah, sorry -- I was confused by the today's auto-redirect to the GCP environment, and the env file you sent me. They're 2 separate concerns. Nothing to change here.